homeassistant/components/vacuum/__init__.py

```diff
@@ -57,6 +57,8 @@
 STATE_PAUSED = STATE_PAUSED
 STATE_RETURNING = 'returning'
 STATE_ERROR = 'error'
+STATE_OFFLINE = 'offline'
+STATE_CHARGING_COMPLETE = 'charging complete'
```

Comment:
We're not allowed to add states in the base vacuum component without first approval in an architecture issue in our architecture repo.

Only states that are commonly used should be added, is the guideline.